web/src/server/api/routers/scores.ts

```diff
@@ -293,6 +293,32 @@ export const scoresRouter = createTRPCRouter({
         },
       });
     }),
+  getNamesList: protectedProjectProcedure
```

Comment:
if this is not too slow for large projects, it is a good idea to reuse this function across tables and have no time filter on it